apps/veil/src/pages/tournament/ui/previous-epochs.tsx

```diff
@@ -84,26 +84,27 @@ const PreviousEpochsRow = observer(
             </Tooltip>
           )}
         </TableCell>
-        {connected && (rewardsLoading || reward !== undefined) && (
-          <TableCell cell loading={isLoading || rewardsLoading}>
```

Comment:
found a rendering bug that was hiding a cell in a certain condition

Reply:
This illustrates my gripe against nesting the loading condition inside of the component, versus making a top level decision.